vms/platformvm/service.go

```diff
@@ -1689,7 +1689,7 @@ func (s *Service) GetTimestamp(_ *http.Request, _ *struct{}, reply *GetTimestamp
 
 // GetValidatorsAtArgs is the response from GetValidatorsAt
 type GetValidatorsAtArgs struct {
-	Height   avajson.Uint64 `json:"height"`
+	Height   avajson.Height `json:"height"`
```

Comment:
I guess this is to prevent breaking change for that args. However I wonder if a separate bool flag or even using negative values would make this more straightforward.

Reply:
Yeah indeed, that + custom parsing is to allow either json field. The idea was that eth supports this kind of thing already in [eth_getBlockByNumber](https://www.quicknode.com/docs/ethereum/eth_getBlockByNumber)

Reply:
But happy to hear the votes either way!

Reply:
It's probably better to have backwards compatibility, so I'm fine with this or using negative values. Should we also add `latest` here?